tests/snapshots/framework-detection.test.js.md

```diff
@@ -9,14 +9,13 @@ Generated by [AVA](https://avajs.dev).
 > Snapshot 1
 
     `◈ Netlify Dev ◈␊
-    ◈ No app server detected and no "command" specified␊
-    ◈ Using current working directory␊
-    ◈ Unable to determine public folder to serve files from␊
+    ◈ No app server detected. Using simple static server␊
+    ◈ Unable to determine public folder to serve files from. Using current working directory␊
     ◈ Setup a netlify.toml file with a [dev] section to specify your dev server settings.␊
     ◈ See docs at: https://cli.netlify.com/netlify-dev#project-detection␊
```

Comment:
I still need to go over `https://cli.netlify.com/netlify-dev` to make sure it makes sense